### PR TITLE
fix: i18n config

### PR DESCRIPTION
**Pull request by stephenjason89**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Related to #7034
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=13rFRPVKjJzQXVC8ZqHZv5YMmwmk4MU7z4HeYk218hEMpQXH&usdamount=100&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

File: utils/config/i18n.ts [hunk 1]
```diff
@@ -25,42 +25,16 @@ export const langsFlags = [
...
-export default () => {
-  return {
-    locale: process.env.VUE_APP_I18N_LOCALE || 'en',
-    fallbackLocale: process.env.VUE_APP_I18N_FALLBACK_LOCALE || 'en',
-    silentTranslationWarn: true,
-    modifiers: {
-      // md: (str) => md.renderInline(str),
-      common: (str) => str.split('.').reduce((o, i) => o[i], commonData),
-    },
-    messages: loadLocaleMessages(),
```
Comment (roiLeo) [inline utils/config/i18n.ts:64]: What about this? it's needed to parse some FAQ content

**Review by roiLeo — COMMENTED** [review]
Hello @stephenjason89 please make pull request on `nuxt` branch in main repo

File: utils/config/i18n.ts (hunk 1 above)
Comment (stephenjason89) [reply utils/config/i18n.ts:64]: Ok made this work on my new pr. Replaced webpacks require.context with import.glob from vite

Comment (stephenjason89) [reply utils/config/i18n.ts:64]: #7045
